app/src/main/java/org/mozilla/gecko/search/SearchWidgetProvider.kt

```diff
@@ -176,6 +176,11 @@ class SearchWidgetProvider : AppWidgetProvider() {
                 )?.toBitmap()
             )
         }
+        val appName = context.getString(R.string.app_name)
```

Comment:
Let's add a new line before this to visually separate the if statement block of work from this new block of work to set the content description